html/pages/settings/webui.inc.php

```diff
@@ -24,6 +24,10 @@
             'svg' => 'svg',
         ),
     ),
+    array('name' => 'webui.graph_stacked',
```

Comment:
You've used two variables now for this. I'd actually say `graphs.stacked` is better but I realise I used `webui.graph_type` for svg so I'd be fine with keeping this as well.